src/WinRT.Runtime/ActivationFactory.cs

```diff
@@ -197,8 +197,27 @@ public static unsafe (ObjectReference<I> obj, int hr) GetActivationFactory<I>(st
 #endif
     static class ActivationFactory
     {
+#if NET

```

Comment:
And to confirm, all of this is guarded for NET to not worry about the vftable types in the .NET standard scenario?

Reply:
Yes and because tests almost always target a specific framework anyway.